src/app/topic/_tests_/topic.spec.js

```diff
@@ -67,7 +67,7 @@ describe('Topic test suite', () => {
         .post(`${baseUrl}/topics`)
         .set('Authorization', adminUser.token)
         .send(requestObject);
-      const errorMessage = 'Technology blockchain does not exist';
+      const errorMessage = 'Technology with name: blockchain does not exist';
```

Comment:
@ebenezerdon why add `with me` to the error message?

Reply:
I added a `with ${paramKey}:` clause to the error message in the `referencedParamValidator` to make it more descriptive.